bedrock/mozorg/templates/mozorg/about/policy/transparency/jan-jun-2017.html

```diff
@@ -80,6 +80,8 @@ <h2 class="section-title" data-accordion-role="tab">{{ _('Government Demands for
         </tbody>
       </table>
 
+      <p>* Amended on <time datetime="2017-09-29">September 29, 2017</time> to include 1 subpoena received in May 2017 to which we did not produce data.</p>
```

Comment:
Let's put this in the footnotes section below and make it the new `footnote-1`, changing the other one to `footnote-2`.

Reply:
Also, even though this page isn't currently localized, we should probably enclose this string for consistency.